torch/csrc/generic/methods/TensorRandom.cwrap

```diff
@@ -240,15 +240,35 @@
   variants:
     - function
   options:
-    - cname: standard_gamma_with_grad
+    - cname: standard_gamma_fwd
       arguments:
         - arg: THTensor* output
           output: True
         - arg: THGenerator* generator
           default: THPDefaultGenerator->cdata
           kwarg_only: True
         - THTensor* alpha
-        - THTensor* grad
+        - THTensor* saved_u
+        - THTensor* saved_x
+]]
+
+[[
+  name: standard_gamma_bwd
+  types:
+    - floating_point
+  backends:
+    - CPU
+  return: argument 0
+  variants:
+    - function
+  options:
+    - cname: standard_gamma_bwd
+      arguments:
+        - arg: THTensor* output
+          output: True
+        - THTensor* alpha
+        - THTensor* saved_u
```

Comment:
It would be nice if we could skip these outputs if we knew an op won't be differentiated. But it's fine as is, and we can fix that later.

Reply:
Already done :-) `standard_gamma()` is a simplified version of `standard_gamma_fwd()` where the unused outputs are discarded.